### DISCUSSION TITLE
Blog Week 機能

### DISCUSSION THREAD

**Discussion by kyouheicf** [increments/qiita-discussions#430] [要望・アイデア・不具合]
例えば Cloudflare では Innovation Week という形で、定期的に Blog Week でまとめて新機能などの発表をおこなっています。
企業活動において、マーケティングの一環にもなっており、また、まとめて技術情報を発信することで盛り上がりやモーメンタムの演出にも繋がります。

> Cloudflare Innovation Weeks 2021
> https://blog.cloudflare.com/2021-innovations-weeks/

Qiita だと Advent Calendar 機能がありますが、12月の単月しか実施できない仕様だと認識しており、非常にもったいないと感じています。
Advent Calendar 機能をベースにして、それを各 Organization が自由に Week 単位で企画できるような柔軟性を展開できると Qiita と Organization の双方にとって、より Win-Win なプラットフォームにできると思い、要望を上げさせていただきます。
個人としても単発でブログをいくつか投稿するより、まとめて Blog Week のようなシリーズで投稿できる機能があると、開発スプリントとしてドライブをかけられますし、ブランディング効果も高いと感じます。

**Comment by tanaka-asami415** [comment]
@kyouheicf 
ありがとうございます！
参考にさせていただきます🙏
